validators/errors.js

```diff
@@ -0,0 +1,22 @@
+export class InexistentItem extends Error {
+  constructor(message = '', ...args) {
+    super(message, ...args);
+    this.message = message;
+  }
+}
+
+export class WrongCredential extends Error {
+  constructor(message = '', ...args) {
+    super(message, ...args);
+    this.message = message;
+  }
```

Comment:
You can give a default response code for each type of error and in the router where you catch the error you can return that response code and like this you dont need to check if the error is an instance of some error and then set the response code based on that.